src/test/java/com/placelab/tests/LoginTestPositive.java

```diff
@@ -0,0 +1,77 @@
+package com.placelab.tests;
+
+import com.placelab.utils.WebDriverSetup;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class LoginTestPositive {
+    private WebDriver driver;
+
+    @Parameters("browser")
+    @BeforeTest
+    public void setup(final String browser) {
+        driver = WebDriverSetup.getWebDriver(browser);
+        driver.get("https://demo.placelab.com/");
+
+        System.out.println("Opened browser: " + browser);
+    }
+
+    @Parameters({"email", "password"})
+    @Test
+    public void testPositiveLogin(final String email, final String password) {
+        final String actualPageTitle = driver.getTitle();
+        final String expectedPageTitle = "PlaceLab";
+        final String expectedUserRole = "Group Admin";
+        final String expectedSignOutButtonText = "Sign out";
+
+        Assert.assertEquals(actualPageTitle, expectedPageTitle);
+        System.out.println("Page title: " + actualPageTitle);
+
+        final boolean isHeaderDisplayed = driver.findElement(By.cssSelector("div#login > p.headline")).isDisplayed();
+        Assert.assertTrue(isHeaderDisplayed);
+
+        final String headerText = driver.findElement(By.cssSelector("div#login > p.headline")).getText();
+        System.out.println("Header is displayed with the following text: " + headerText);
```

Comment:
you should usually avoid committing the print outs, they are useful when you're developing tests, but not necessary for the final results